ngraph/python/tests/test_ngraph/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+import ngraph as ng
+
+if sys.platform == "win32":
+    # ngraph.dll is installed in PYTHON_INSTALL_DIR/Lib as default
+    # (which is 2 directories above ngraph module)
+    # and this path needs to be visible to the _pyngraph module.
+    #
+    #
+    # If you're using a custom installation of nGraph,
+    # add the location of ngraph.dll to your system PATH.
+    ngraph_dll_dir = os.path.join(os.path.dirname(ng.__file__), ".", ".")
```

Comment:
Shouldn't it be `..` twice instead of single dots?

Reply:
interesting, single dot works on win too, but you are right ".." is more clear

Reply:
@mbencer Are you sure "." were working? It shouldn't be possible in any case.

Reply:
I think the `import ngraph as ng` statement in this file does the trick. It imports the whole ngraph module, and the `__init__.py` file for that module does `from ngraph.impl import Node`. The `ngraph.impl` module sets the correct paths.

I think we can even remove the whole if statement from `test_ngraph/__init__.py` and it should still work.

Reply:
@jiwaszki you are right, as @tomdol mentioned alone `import ngraph` causes that ngraph.dll dir path is added to `PATH` so it is the reason that is works for me (single dot is not correct). Content of if statement is not needed.